portal-services/src/main/resources/portal-services-openapi.yaml

```diff
@@ -10403,6 +10403,31 @@ paths:
                 $ref: '#/components/schemas/PortalUser'
         500:
           $ref: '#/components/responses/GenericApiError'
+          
+  /portal/portalUser/customerIdsForUsername:
+    get:
+      tags:
+      - Portal Users
+      summary: Get customerIds for username
+      operationId: getCustomerIdsForUsername
+      parameters:
+      - name: username
+        in: query
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: integer
```

Comment:
this should match the return type from PortalUserPortalController, which is a list of PortalUsers